pages/SharedRingDB.qml

```diff
@@ -112,25 +112,11 @@ Rectangle {
         MoneroComponents.LabelSubheader {
             Layout.fillWidth: true
             textFormat: Text.RichText
-            text: "<style type='text/css'>a {text-decoration: none; color: #FF6C3C; font-size: 14px;}</style>" +
-                  qsTr("Outputs marked as spent") + " <a href='#'>" + qsTr("Help") + "</a>" + translationManager.emptyString
-            onLinkActivated: {
-                sharedRingDBDialog.title  = qsTr("Outputs marked as spent") + translationManager.emptyString;
-                sharedRingDBDialog.text = qsTr(
-                    "In order to obscure which inputs in a Monero transaction are being spent, a third party should not be able " +
-                    "to tell which inputs in a ring are already known to be spent. Being able to do so would weaken the protection " +
-                    "afforded by ring signatures. If all but one of the inputs are known to be already spent, then the input being " +
-                    "actually spent becomes apparent, thereby nullifying the effect of ring signatures, one of the three main layers " +
-                    "of privacy protection Monero uses.<br>" +
-                    "To help transactions avoid those inputs, a list of known spent ones can be used to avoid using them in new " +
-                    "transactions. Such a list is maintained by the Monero project and is available on the getmonero.org website, " +
-                    "and you can import this list here.<br>" +
-                    "Alternatively, you can scan the blockchain (and the blockchain of key-reusing Monero clones) yourself " +
-                    "using the monero-blockchain-mark-spent-outputs tool to create a list of known spent outputs.<br>"
-                ) + translationManager.emptyString
-                sharedRingDBDialog.icon = StandardIcon.Information
-                sharedRingDBDialog.open()
-            }
+            text: qsTr("Outputs marked as spent") + translationManager.emptyString
```

Comment:
I would revert this back to the same `qsTr`, otherwise all this huge strings have to be retranslated for no reason.